RELEASE.md

```diff
@@ -50,7 +50,13 @@ After electing the Release Manager for releasing a new Codacy Chart version:
 
 -   [ ] 4.  Inform all stakeholders outside the engineering team that a new release proccess is being started (tag @here in the Slack channel [#sh_releases](https://codacy.slack.com/channels/sh_releases)).
 
-## 2. Preparing the release for testing
+## 2. Before starting the release
+
+Before starting the self-hosted release process, the Release Manager should ensure that all the features that should be introduced for this release were successfully deployed and passed all CI workflows. Also, it is recommended to check the Jira tickets to see if there are any red flags that might hinder the release process or cause any delays down the line.
+
+The Release Manager should also confirm that the changes were properly tested before proceeding with the release. If the Release Manager finds any problems during this process, the Squads involved must be informed to find possible solutions to fix the issues before the release goes forward.
+
```

Comment:
It's great that you're giving explicit visibility to all these validations and checks @MiguelMarcelino! :muscle: Typically, these go a bit under the radar and as you mention it can cause trouble further down the line.

However, I think these validations should even be performed at an earlier stage, more specifically this is what should be validated when we mention "getting a status from each team" and "confirming if the release can go ahead" on [lines #L37-L39 above](https://github.com/codacy/chart/pull/777/files#diff-2b1b69303b927a484e02c7fad9fc87d0d3ff0dc22ae1da0ecd0dc935d922a23cR37-R39).